rest-openehr/src/main/java/org/ehrbase/rest/openehr/OpenehrCompositionController.java

```diff
@@ -335,38 +342,19 @@ public ResponseEntity deleteComposition(
         }
     }
 
-    /**
-     * Acts as overloaded function and calls the overlapping and more specific method
-     * getCompositionByTime. Catches both "/{ehr_id}/composition/{version_uid}" and
-     * "/{ehr_id}/composition/{versioned_object_uid}" which works because their processing is the
-     * same. "{?version_at_time}" is hidden in swagger-ui, it only is here to be piped through.
-     */
-    @GetMapping("/{ehr_id}/composition/{version_uid}")
```

Comment:
Seems like this method is used for documenting the different parameter types of supported for getting contribution. I think we should keep it

Reply:
The new version of spring doesn't accept this syntax: "{?version_at_time}" 
The new version should be identical